apps/omg/lib/omg/state.ex

```diff
@@ -92,9 +96,11 @@ defmodule OMG.State do
   def init(:ok) do
     {:ok, height_query_result} = DB.get_single_value(:child_top_block_number)
     {:ok, child_block_interval} = Eth.RootChain.get_child_block_interval()
+    fee_claimer_address = Application.get_env(:omg, :fee_claimer_address, @no_fee_claimer_address)
```

Comment:
if it's not to big of a burden, the proper pattern is to pass the configuration values from the supervisor down, not read it in the configuration.